plugin/src/main/java/io/snyk/eclipse/plugin/views/SnykView.java

```diff
@@ -138,12 +133,17 @@ private void createColumns() {
 
   private Image findSeverityImage(DisplayModel model) {
     String severity = model.severity;
-    if (severity == null) return null;
-
-    if (severity.equalsIgnoreCase("critical")) return CRITICAL_SEVERITY;
-    if (severity.equalsIgnoreCase("high")) return HIGH_SEVERITY;
-    if (severity.equalsIgnoreCase("medium")) return MEDIUM_SEVERITY;
-    if (severity.equalsIgnoreCase("low")) return LOW_SEVERITY;
+    if (severity == null)
```

Comment:
Let's have editorconfig and format everything accordingly eventually. This'd save some review times.

Reply:
We have an .editorconfig. Apparently Eclipse & IntelliJ interprete it differently though :(. https://github.com/snyk/snyk-eclipse-plugin/blob/9e320a5cc740021d1813d31d8a00505b57db0a3e/.editorconfig. I'll add a ticket to Jira though, to try to resolve it.